R/model.R

```diff
@@ -1024,7 +1029,23 @@ optimize_method <- function(data = NULL,
                             algorithm = NULL,
                             init_alpha = NULL,
                             iter = NULL,
-                            sig_figs = NULL) {
+                            sig_figs = NULL,
+                            threads = NULL) {
```

Comment:
Not so sure about the name here. but `threads_per_chain` does not work for optimize/variational.

Reply:
Yeah that's a good question. Ideally we would use the same argument name but you're right that there are no chains here. I guess `threads` is fine. @mitzimorris or @bbbales2 any thoughts on what name we should use here?